### ISSUE TITLE
[wasm] Launcher.LaunchUriAsync page is suspending

### ISSUE THREAD

**Issue by lld1995** [unoplatform/uno#11936]
### Current behavior

like Launcher.LaunchUriAsync("https://www.google.com") invoked,chrome current page can't click anything

### Works on UWP/WinUI

None

**Comment by agneszitte** [comment]
@jeromelaban I am able to reproduce the issue and here is the info so far:
- **Package versions**: Latest stable [4.8.24](https://www.nuget.org/packages/Uno.UI/4.8.24) but also with latest dev [4.9.0-dev.555](https://www.nuget.org/packages/Uno.UI/4.9.0-dev.555)
- **Affected Platform**:  Only WASM with Google Chrome (Tested with Version 111.0.5563.147 (Official Build) (64-bit))
_Issue not happening with Microsoft Edge (Tested with Version 112.0.1722.39 (Official Build) (64-bit))_
- **Sample App for the repro**: (using latest dev [4.9.0-dev.555](https://www.nuget.org/packages/Uno.UI/4.9.0-dev.555))
[LaunchUriAsyncTest.zip](https://github.com/unoplatform/uno/files/11225336/LaunchUriAsyncTest.zip)
- **Repro Steps**: 
   - Launch the app
   - Click on the Twitter button to open the first link
   - Twitter page opening on a separate tab in the browser
   - Go back to the previous browser tab where the app is opened
   - Try to click on the Google button to open the second link or try to click again on the previous Twitter button
     - Notice the buttons are not clickable anymore
     (If you try to add another control to the MainPage like a CheckBox for example, when redoing the same steps, the CheckBox also will not be clickable anymore like the other buttons)

**Comment by jeromelaban** [comment]
@agneszitte it may be a dev issue. Can you try with a canary build of Chrome?

**Comment by agneszitte** [comment]
@jeromelaban you are right, seems to be a temporary issue for Google Chrome as I am not able to reproduce the issue anymore when testing with _Google Chrome Version 114.0.5712.0 (Official Build) **canary** (64-bit)_
